setup.cfg

```
@@ -47,7 +47,8 @@ doc =
 test =
     katsdpsigproc[CUDA]
     async-solipsism
-    pytest
+    async-timeout
+    pytest>8
     pytest-asyncio>=0.18,<0.22
```

I'm a bit torn. To avoid anyone else getting the cryptic incompatibilities, I'd consider
```suggestion
    pytest-asyncio>=0.21.2,<0.22
```
On the other hand if pytest 8.0.0 and pytest-asyncio 0.18 work together then this could be left as is. But I don't think it's worth spending the time to test that combination.

---

Fair points all around. Will update to set the lower bound at 0.21.2